src/Packetery/Module/WpdbAdapter.php

```diff
@@ -325,7 +325,7 @@ public function getLastWpdbError(): string {
 	 * @return \Generator
 	 */
 	public function getWpdbQueries(): \Generator {
-		if ( isset( $this->wpdb->queries ) ) {
+		if ( count( $this->wpdb->queries ) > 0 ) {
```

Comment:
`TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /app/wcbeta/wp-content/plugins/packeta/src/Packetery/Module/WpdbAdapter.php:328`

Reply:
Dobrý catch